namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/s3.tf

```
@@ -0,0 +1,34 @@
+/*
+ Based on https://github.com/ministryofjustice/cloud-platform-terraform-s3-bucket/tree/main/example
+ */
+module "interventions_s3_bucket" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=4.6"
+
+  team_name              = var.team_name
+  business-unit          = "Digital and Technology"
+  application            = var.application
+  infrastructure-support = var.infrastructure_support
+
+  is-production    = var.is_production
+  environment-name = var.environment
+  namespace        = var.namespace
+
+  providers = {
+    # Can be either "aws.london" or "aws.ireland"
+    aws = aws.london
+  }
+}
+
+resource "kubernetes_secret" "interventions_s3_bucket" {
+  metadata {
+    name      = "interventions-s3-bucket"
```

As this secret will be retrieved via `kubectl get secret/interventions-s3-bucket --namespace=hmpps-interventions-dev`, `interventions` seems redundant. What do you think about?
```suggestion
    name      = "storage-s3-bucket"
```

---

works for me 👍